README.md

```diff
@@ -1,15 +1,127 @@
+
+* Source Code Policy
+  * To provide feedback on the [Federal Source Code Policy](https://sourcecode.cio.gov/), you should follow [this issue tracker](https://github.com/WhiteHouse/source-code-policy/issues)
+
+* Code.gov
+    * To provide feedback on code-gov-front-end, please checkout our [Contributing Guildelines](CONTRIBUTING.md).
+    * To contribute to the Code.gov style guide, head over to the [code-gov-style](https://github.com/GSA/code-gov-style) repo.
+    * Checkout [code-gov](https://github.com/GSA/code-gov) for a list of additional project repositories. If you aren't sure where your question or idea fits, this is a good place to share it.
+
+## Questions?
+
+If you have questions, please feel free to [send us an email](mailto:code@gsa.gov).
+
+## Getting Started
+
+After you have cloned this repo, you can use `npm install` to install all of the
+project’s dependencies.
+
+You can then run the server using `npm run start`.
+
+By default, the development server will listen on <http://localhost:8080/>. You can change the default port by setting the `PORT` environment variable before starting the server (for example, `PORT=3000 npm start`).
+
+## Testing
+### Unit tests
+This app uses...
+
+### End-to-end tests
+End-to-end testing is done with...
+
+## Deployment
+This app uses the `github-deploy` package for handling deployment. To configure
+deployment, customize the `config/github-deploy` and `webpack.github-deploy`
+files to match your settings. When ready to deploy, run
+`npm run federalist-deploy:dev` or `npm run federalist-deploy:prod`, depending on your
+intended destination.
+
+If you are deploying from a fork, you will have to set
+the `GIT_REMOTE_NAME` environmental variable,
+like `GIT_REMOTE_NAME="upstream" npm run federalist-deploy`.
+
+## Deploying Arbitrary Branch
+If you'd like to deploy an arbitrary branch, set the `GIT_BRANCH_NAME` environmental variable.  For example the following code will deploy the current branch to https://federalist-proxy.app.cloud.gov/preview/gsa/code-gov-web/federalist-demo/#/
+```
+GIT_BRANCH_NAME="federalist-demo" npm run federalist-deploy
+```
+
+
+## Specifying API URL
+The app uses the production API by default.  To use the staging API,
+set an API_URL environmental variable for the npm process. Here's two examples:
+```
+API_URL=stag npm run start
+API_URL=staging npm run start
+API_URL='https://code-api-staging.app.cloud.gov/api/' npm run start
+API_URL=staging GIT_REMOTE_NAME=upstream npm run federalist-deploy:dev
+```
+
+## Specifying an API Key
+The app uses the API key provided in the code-gov-config.json by default.
+If you want to override that, specify an `CODE_GOV_API_KEY` environmental variable.  Here's an example:
 ```
-npm run start
+CODE_GOV_API_KEY=l87sfdi7ybc2bic7bai8cb2i176c3b872tb3 npm run start
 ```
 
-# deployment
-Read about how to publish to Github pages, Federalist and elsewhere [here](DEPLOYMENT.md)
+## Deployment Problems
```

Comment:
Could you remove the section on Deployment Problems? I fixed this with `ghpages.clean()` in https://github.com/GSA/code-gov-front-end/blob/master/deploy.js#L25